sechub-pds/src/main/resources/application-pds_integrationtest.yml

```diff
@@ -2,6 +2,9 @@
 spring.profiles.include:
 - pds_localserver
 
+server:
```

Comment:
This should be set for localserver as well.

Reply:
### General
Are you talking about SecHub server or PDS ?

I did explicit set loopback to integration tests only because adding to local server profile  This would make the server not working per default!

### Integration test profile situation
The integration test profile has default passwords and that's the reason why we do NOT want to have the server exposed.

### Differ between PDS SecHub
#### PDS
When somebody uses a PDS it's 99% not on the same server except in our integration tests  PDS "implementations" are normally running inside a docker container, docker compose or in k8s. Defining here a "127.0.0.1" would bring up problems / more configuration. The docker run call

```
docker run --rm -p 127.0.0.1:$exposed_port:$container_port/tcp $env_data --name $container_name $image_name
```
should protect the PDS instance.

RESULT: --> for PDS I would not change the behaviour.

### SecHub
For testing purposes etc. it can happen that a developer starts the server with custom / dedicated profiles. But... when not using integration test profile the developer MUST define a password and there are no hardcoded passwords . 

Yes, the server will be visible in network. But we assume that our server software is secure

RESULT: --> for SecHub I would not change the behaviour.